library/commandline/src/modules/CommandLine.rb

```diff
@@ -1276,9 +1276,9 @@ def GetInput(prompt, type)
 
       res = if type == :nohistory
               Convert.to_string(SCR.Read(path(".dev.tty.nohistory")))
-            elsif type == :noecho
+      elsif type == :noecho
               Convert.to_string(SCR.Read(path(".dev.tty.noecho")))
-            else
+      else
```

Comment:
I am not sure if this is better, now it looks like relates to res assignment and not to if condition. so :-1: